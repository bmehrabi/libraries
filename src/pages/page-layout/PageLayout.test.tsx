import React from 'react';
import {render} from '@testing-library/react';
import PageLayout from './PageLayout';
import {BrowserRouter} from 'react-router-dom';

test('matches the snapshot', () => {
    const {container} = render(
      <BrowserRouter>
          <PageLayout />
      </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
});
