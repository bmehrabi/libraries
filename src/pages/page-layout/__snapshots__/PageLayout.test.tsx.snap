// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-k008qs"
  >
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionAbsolute css-n7rtur-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-m9mp0w-MuiToolbar-root"
      >
        <button
          aria-label="open drawer"
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart MuiIconButton-sizeMedium css-1johsky-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="MenuIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3zm0-5h18v-2H3zm0-7v2h18V6z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <h1
          class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-uh02xs-MuiTypography-root"
        >
          Wire App
        </h1>
      </div>
    </header>
    <div
      class="MuiDrawer-root MuiDrawer-docked css-cvumne-MuiDrawer-docked"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiDrawer-paper MuiDrawer-paperAnchorLeft MuiDrawer-paperAnchorDockedLeft css-12i7wg6-MuiPaper-root-MuiDrawer-paper"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1dd0suk-MuiToolbar-root"
        >
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="ChevronLeftIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 7.41 14 6l-6 6 6 6 1.41-1.41L10.83 12z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <nav
          class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
        >
          <a
            href="/"
          >
            <div
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="SearchIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-tlelie-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                >
                  Search
                </span>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </div>
          </a>
          <a
            href="/development-process"
          >
            <div
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="CodeIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M9.4 16.6 4.8 12l4.6-4.6L8 6l-6 6 6 6zm5.2 0 4.6-4.6-4.6-4.6L16 6l6 6-6 6z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-tlelie-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                >
                  Development process
                </span>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </div>
          </a>
          <a
            href="/about"
          >
            <div
              class="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
              role="button"
              tabindex="0"
            >
              <div
                class="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="InfoIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 15h-2v-6h2zm0-8h-2V7h2z"
                  />
                </svg>
              </div>
              <div
                class="MuiListItemText-root css-tlelie-MuiListItemText-root"
              >
                <span
                  class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                >
                  About Me
                </span>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </div>
          </a>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth css-1b008ma-MuiDivider-root"
          />
        </nav>
      </div>
    </div>
    <main
      class="MuiBox-root css-fxbtpg"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      />
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 css-kks9oq-MuiPaper-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-16z8frn-MuiTypography-root"
        >
          Developed at Mar. 2024 
        </p>
      </div>
    </main>
  </div>
</div>
`;
